### PR TITLE
Updated container-selinux url to point to the right path

### PR DESCRIPTION
**Pull request by igarcia-sugarcrm**

### CODE REVIEW

File: nodeup/pkg/model/docker.go [hunk 1]
```diff
@@ -557,7 +557,7 @@ var dockerVersions = []dockerVersion{
 		ExtraPackages: map[string]packageInfo{
 			"container-selinux": {
 				Version: "2.68",
-				Source:  "http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.68-1.el7.noarch.rpm",
+        Source:  "http://mirror.centos.org/centos/7.6.1810/extras/x86_64/Packages/container-selinux-2.68-1.el7.noarch.rpm",
```
Comment (rifelpet) [inline nodeup/pkg/model/docker.go]: It looks like your editor may have replaced the tabs with spaces. Can you update your commit to use tabs?

Comment (mikesplain) [reply nodeup/pkg/model/docker.go]: I pushed a fix, so this should be resolved shortly.

**Comment by mikesplain** [comment]
I believe this is the better fix for this issue since it leaves versions of packages the same, which are certified per K8s version.

File: nodeup/pkg/model/docker.go (hunk 1 above)
Comment (igarcia-sugarcrm) [reply nodeup/pkg/model/docker.go]: Thanks! Sorry about that.

**Comment by mikesplain** [comment]
Tested and confirmed this fixes the issue at hand. Thanks for the fix everyone! We'll get cherry-picks open asap.

/lgtm
/approve

**Pull request merged**